dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/service/SoftwareService.java

```diff
@@ -177,7 +177,7 @@ private String getRepositoryUrl(String artifactId) {
 
   public class Version implements Comparable<Version> {
 
-    private String version;
+    public String version;
 
     @JsonValue
     public final Map<String, String> getJson() {
```

Comment:
Using `@JsonValue` on just the string was behaving oddly:  it output a dictionary of the string index and the character at that index.  Using the decorator on a map output behaved more sensibly.